### ISSUE TITLE
Custom dataset training

### ISSUE THREAD

**Issue by Zhihang-Li** [soushirou/stylegan3-encoder#2]
Thank you for your great work.
Using the official implement of Stylegan3, we successfully train a generator on our custom dataset.
However, when training the encoder with your code, I can't obtain a promising result. Here are the results of mine:
![image](https://user-images.githubusercontent.com/28439806/147024533-9a8d2132-b543-44ae-ae20-545e9f44db78.png)
![image](https://user-images.githubusercontent.com/28439806/147024652-8458c336-ba70-42b9-bfe5-c9cc16579eec.png)
Have you ever encountered such a situation？

**Comment by soushirou** [comment]
l2 loss learning curve seems like my experiment's 
The most recent successful training configuration is follow
{
  "model_architecture": "base",
  "dataset_dir": "data/ffhq",
  "num_gpus": 8,
  "batch_size": 32,
  "batch_gpu": 4,
  "generator_pkl": "pretrained/stylegan3-t-ffhq-1024x1024.pkl",
  "w_avg": true,
  "num_encoder_layers": 1,
  "val_dataset_dir": null,
  "training_steps": 100001,
  "val_steps": 10000,
  "print_steps": 50,
  "tensorboard_steps": 50,
  "image_snapshot_steps": 100,
  "network_snapshot_steps": 5000,
  "learning_rate": 0.001,
  "l2_lambda": 1.0,
  "lpips_lambda": 0.8,
  "id_lambda": 0.1,
  "reg_lambda": 0.0,
  "gan_lambda": 0.0,
  "edit_lambda": 0.0,
  "random_seed": 0,
  "num_workers": 3,
  "resume_pkl": null,
  "run_dir": "exp/config-b/00000-base-ffhq-gpus8-batch32"
}
you can use "train_config_b.sh" 
and I haven't experimented with custom datasets and only nvidia's official generator weight(stylegan3-t) are used.
